wp-content/themes/goonj-crm/assets/styles/lg.scss

```diff
@@ -58,6 +58,21 @@
         .error {
             font-size: 14px;
         }
+
+        .af-container:not(:has(.af-container)) {
+            display: flex !important;
+            flex-wrap: wrap !important;
+        }
```

Comment:
My understanding is: in our layout we want certain container elements to use a flexbox layout, but only if they do not have nested containers of the same class. This ensures that the outer containers behave as flex containers, while nested containers can have different styling or layout rules applied.

Reply:
Yes, that it correct